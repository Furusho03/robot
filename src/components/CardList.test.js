import { shallow } from 'enzyme';
import React from 'react';
import CardList from './CardList';

import toJson from 'enzyme-to-json';
// import TestRenderer from 'react-test-renderer';

// const CardRender = TestRenderer.create(
//     <Card />
// );

it('test render', () => {
    const mockRobots = [
        {
            id: 1,
            name: 'Jhon Snow',
            username: 'Jhon',
            email: 'jhon@gmail.com'
        }
    ]
    expect(toJson(shallow(<CardList robots={mockRobots} />))).toMatchSnapshot();
})