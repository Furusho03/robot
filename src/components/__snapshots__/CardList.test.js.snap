// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test render 1`] = `
<div>
  <Card
    email="jhon@gmail.com"
    id={1}
    key="0"
    name="Jhon Snow"
  />
</div>
`;
